http/handler.go

```diff
@@ -115,7 +115,7 @@ func respondStandby(core *vault.Core, w http.ResponseWriter, reqURL *url.URL) {
 	redirectURL := url.URL{
 		Scheme:   advertiseURL.Scheme,
 		Host:     advertiseURL.Host,
-		Path:     reqURL.Path,
+		Path:     advertiseURL.Path + reqURL.Path,
```

Comment:
@vespian after some consideration -- if you want to add 10 slashes to the end of your advertise_addr, I'm not going to strip them for you. respondStandby is a pretty dumb pass-through function, so I don't want to touch its inputs any more than needed.